dbt_schema_builder/templates/model_sql_pii.tpl

```diff
@@ -1,10 +1,16 @@
 -- This file is automatically generated. Do not update by hand.
 
 {{ '{{' }} config(schema='{{app}}_PII', alias='{{relation.alias}}') {{ '}}' }}
-
+{% set soft_del_ns = {'found':false} %}
 SELECT
 {% for col in relation.columns %}
     {{ col.name|upper|indent }}
     {{- ", " if not loop.last }}
+    {%- if col.name|upper == raw_schema.soft_delete_column_name|upper -%}
+        {%- if soft_del_ns.update({'found':true}) -%}{% endif %}
```

Comment:
I get what's going on in this file, but why is this line an `if`. Seems like you just want to update the dict, no?

Reply:
tl;dr: Jinja is a pain and this is actually necessary. Longer: https://stackoverflow.com/questions/9486393/jinja2-change-the-value-of-a-variable-inside-a-loop